player/clipboard/clipboard.c

```diff
@@ -40,7 +40,7 @@ static const struct clipboard_backend *const clipboard_backend_list[] = {
 #if HAVE_COCOA
     &clipboard_backend_mac,
 #endif
-#if HAVE_WAYLAND_PROTOCOLS_1_39
+#if HAVE_WAYLAND && HAVE_WAYLAND_PROTOCOLS_1_39
```

Comment:
Why? Is it possible to have protocols without wayland?

Reply:
`HAVE_WAYLAND_PROTOCOLS_*` isn't defined if wayland isn't available, that's why the feature check inside toplevel meson.build was there. On master, "#define HAVE_WAYLAND_PROTOCOLS_1_39 0" is even on no wayland builds. We would need the wayland check here anyway, once we bump w-p to 1.39 or above.